benchmarking/benchmark.py

```diff
@@ -29,11 +36,22 @@ def main():
     parser.add_argument('-t', '--threshold', type=float, default=0.0001,
                         help='Value of the standard deviation trigger on '
                         'eigenvalue.')
+    parser.add_argument('-o', '--output_name', type=str,
```

Comment:
```suggestion
    parser.add_argument('-o', '--output-name', type=str,
```
The variable will still be called `args.output_name` because hyphens get turned into underscores.